src/components/Card.js

```diff
@@ -1,20 +1,21 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 
 function Card(props) {
   const [saveCardPopup, setSaveCardPopup] = useState(false);
   const [mobileMode, setMobileMode] = useState();
+  const [marked, setMarked] = useState(false);
 
   window.addEventListener('resize', () => {
```

Comment:
`  window.addEventListener('resize', () => {`

you shouldn't add listeners just inside a component's body because if the component rerenders the listener will be added again. You need to add listeners only in `useEffect` and you need to remove the listeners in `clean up` function of the `useEffect`